pkgs/tools/misc/dvtm/unstable.nix

```diff
@@ -1,6 +1,7 @@
 {callPackage, fetchFromGitHub, fetchpatch}:
 callPackage ./dvtm.nix {
-  name = "dvtm-unstable-2018-03-31";
+  pname = "dvtm-unstable";
+  version = "2018-03-31";
```

Comment:
```suggestion
  pname = "dvtm";
  version = "unstable-2018-03-31";
```

Reply:
i think this would be a breaking change. is it wanted?

Reply:
right.